src/indexer/index.ts

```diff
@@ -15,16 +15,17 @@ import {
 } from "@algo-builder/algob/build/types";
 import type { LogicSigAccount } from "algosdk";
 import { getProposalLsig } from "../contract/dao";
+import indexerClient from "../config/indexer.config";
 declare var AlgoSigner: any; // eslint-disable-line
 
 export const searchForAssets = async (
 	tokenName: string
 ): Promise<Record<string, any> | undefined> => {
 	try {
-		const accountInfo = await AlgoSigner.indexer({
-			ledger: CHAIN_NAME,
-			path: `/v2/assets?name=${tokenName}`,
-		});
+		const accountInfo = await indexerClient
```

Comment:
I think we should create Indexer instance in src/indexer. How you think about it? I not sure this is good idea.

Reply:
Yes, The instance is created inside `config/indexer.config`. I think the current path is good because this also includes ports and URL which needed for instance.